apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-docs.component.html

```diff
@@ -1,44 +1,48 @@
-<fd-docs-section-title [id]="'def'" [componentName]="'Radio Button Group'">
-    Radio Button Group with Radio buttons as Content
+<fd-docs-section-title [id]="'projection'" [componentName]="'Radio Button Group'">
+    Radio Button Group With Radio Buttons Passed as Content.
 </fd-docs-section-title>
 
-<description>This example shows Radio Button Group implementaion with Content as Radio Buttons</description>
+<description>
+    Platform Radio Buttons Passed as Content
```

Comment:
Hi there! I'm not sure what you mean by "passed as content"? Do you mean they have text labels?

Reply:
corrected to "passed as content projection". It means that some content going under some other html element.
Ex: 
`<radio-group>
       <radio>radio1</radio>
       <radio>radio2</radio>
</radio-group>
`